extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -192,7 +191,6 @@ export class AmpAnalytics extends AMP.BaseElement {
     if (this.iniPromise_) {
       return this.iniPromise_;
     }
-    toggle(this.element, false);
```

Comment:
If preferred, I could instead keep the toggle if not using iframe transport, by putting this after line 204:
.then(() => if (!this.config_['transport'] || !this.config_['transport']['iframe']) { toggle(this.element, false); }
Let me know which you prefer (or some other option).